mentors/README.md

```diff
@@ -49,6 +49,8 @@ The `link_url` and `link_text` are for linking to a personal site, if you want t
 Note that if you leave a field `null`, we will use the content from your public GitHub profile.
 If you want to leave a field blank, set it to an empty string.
 
+The first line of the file should consist of only a single `[` character and the last line should consist of only a single `]` character.
+
```

Comment:
I think this is a good idea. How about specifying first that the file should contain a JSON array, and then follow that up with the clarification that you have? I think that extra context will make it a bit easier to understand.

Reply:
Thanks for the suggestion.  I agree.

Reply:
There's a small tyypo:

> bio's

This should be bios.

Reply:
Thanks!  Got it.  This is ready to be merged.